#moveByUnits: 0
#moveToUnits: 1
uint8 flag
int16 yaw
int16 roll
int16 pitch
int16 duration
---
bool result 
